### PR TITLE
Polytope constructor

### PR DESCRIPTION
**Pull request by sebasguts**

### CODE REVIEW

File: src/convert.jl [hunk 1]
```diff
@@ -42,7 +42,7 @@ function convert(::Type{Polymake.pm_Matrix{Polymake.pm_Integer}}, matrix::Array{
     for i in 1:rows
         for j in 1:cols
             converted = convert(Polymake.pm_Integer,matrix[i,j])
-            Polymake.set_entry(pm_matrix, i-1, j-1, converted )
```
Comment (kalmarek) [inline src/convert.jl]: You don't need to convert, it's handled in `setindex!`

File: test/runtests.jl [hunk 2]
```diff
@@ -201,4 +201,5 @@ end
     end
 
     include("sets.jl")
+    include("perlobj.jl")
```
Comment (saschatimme) [inline test/runtests.jl]: Did you forget to check that file in?

Comment (sebasguts) [reply test/runtests.jl]: Ah, screw me. Thanks for noticing.

**Comment by sebasguts** [comment]
This is now contained in #23

**Pull request closed without merging**